src/java.base/share/classes/sun/security/util/ManifestEntryVerifier.java

```diff
@@ -207,38 +208,57 @@ public JarEntry getEntry()
             return signers;
         }
 
-        JarConstraintsParameters params =
-            getParams(verifiedSigners, sigFileSigners);
+        CodeSigner[] entrySigners = sigFileSigners.get(name);
+        Map<String, Boolean> permittedAlgs =
```

Comment:
maybe permittedAlgsChecker as variable name ?  the Map contains both permitted and non-permitted algs.

Reply:
yes, algsPermittedStatus sounds better. Thanks.